apache/trafficcontrol
# Migrate dynamic scripted Grafana Dashboards to Scenes

1. Upgrade to Grafana v10.2.3
2. Migrate Scripted Dashboards to Grafana Scenes
- I've transitioned our existing scripted dashboards to the new Grafana Scenes. The migration also ensures compatibility with future Grafana updates and aligns with Grafana's recommended best practices. Check it here: [New in Grafana 10: Grafana Scenes for building dynamic dashboarding experiences
](https://grafana.com/blog/2023/08/03/new-in-grafana-10-grafana-scenes-for-building-dynamic-dashboarding-experiences/)
3. Update documentation: the way to configure Grafana and the current Traffic Portal doesn't use directly the Grafana dashboards

<hr/>

## Which Traffic Control components are affected by this PR?
- Traffic Stats
- CDN in a Box

## What is the best way to verify this PR?

- Build Traffic Stats
- Run CDN in a Box with Traffic Stats, Grafana
- Visit Grafana server, go to Apps -> TrafficControl Grafana Scenes
- See the dashboards

Check the images below

## If this is a bugfix, which Traffic Control versions contained the bug?



## PR submission checklist
